src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,27 +1,33 @@
 <script setup>
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
+import { MaInput } from "@mobileaction/action-kit"
+import { MaBadge } from "@mobileaction/action-kit"
 
+const checkedNumbers = ref([])
+const excludeWords = ref('is, a, an, the') // default words to exclude
 const sentence = ref('')
-const MAX_N_GRAM_COUNT = 3
 const nGrams = ref([])
+const words = computed( () => {
+    const excludeWordsArray = excludeWords.value.split(/\s*,\s*/).map(word => word.toLowerCase())
+    return sentence.value.split(/\s+/).filter(word => !excludeWordsArray.includes(word.toLowerCase()));
+})
 
 const generateNGrams = () => {
```

Comment:
- We can make the operations inside the function more understandable by assigning them to more descriptive variables and adding necessary comments. 

- We can assign the expression words.value used in multiple places to a variable. 

- We can also change the name uniqueCombinations to a more descriptive name.

```
const generateNGrams = () => {
  nGrams.value = [];
  const sortedNGrams = [...selectedNGrams.value].sort((a, b) => a - b);
  const currentWords = filteredWords.value;

  sortedNGrams.forEach(n => {
    const nGramSet = new Set();

    for (let i = 0; i <= currentWords.length - n; i++) {
      const nGramCombination = currentWords.slice(i, i + n).join(' ');
      nGramSet.add(nGramCombination);
    }

    if (nGramSet.size) {
      nGrams.value.push({ n, combinations: Array.from(nGramSet) });
    }
  });
};
```